lib/school.rb

```diff
@@ -0,0 +1,27 @@
+class School
+  attr_reader :start_time,
+              :hours_in_school_day,
+              :student_names
+
+  def initialize(start_time, hours_in_school_day)
+    @start_time = start_time
+    @hours_in_school_day = hours_in_school_day
+    @student_names = []
+  end
+
+  def add_student_name(name)
+    student_names << name
+  end
+
+  def end_time
+    @start_time += @hours_in_school_day
+  end
+
+  def is_full_time(hours_in_school_day)
+    hours_in_school_day >= 4
+  end
+
+  def standard_student_names(student_names)
```

Comment:
Same note as above - the interaction pattern calls for this method to be run with no arguments.